packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -3210,6 +3213,75 @@ namespace Ifpack2 {
       }
     }; 
 
+
+    template<typename MatrixType>
+    struct ReduceMultiVector {
```

Comment:
I thought the point was to fuse the norm with another kernel -- it looks like this is just a norm by itself.

Reply:
My comment may confuse you. Previously the code fuse norm computation and multi vector conversion. In this commit, I first compute diff of values in a multi vector with a conversion. Then, I reduce it. So, here I can say that I fuse solve and diff computation. This results in better usage of team threads.

Reply:
ok